fabric-tags-v0/src/main/java/net/fabricmc/fabric/api/tags/FabricBlockTags.java

```diff
@@ -0,0 +1,10 @@
+package net.fabricmc.fabric.api.tags;
+
+import net.fabricmc.fabric.api.tag.TagRegistry;
+import net.minecraft.block.Block;
+import net.minecraft.tag.Tag;
+import net.minecraft.util.Identifier;
+
+public final class FabricBlockTags{
+	public static final Tag<Block> BOOKSHELF = TagRegistry.block(new Identifier("fabric-tags-v0", "bookshelf"));
```

Comment:
```suggestion
	public static final Tag<Block> BOOKSHELVES = TagRegistry.block(new Identifier("fabric", "bookshelves"));
```

Reply:
Considering previous debates about tag identifier namespaces, I'd say having tags use the module's mod id instead of `fabric` might be better than just `fabric`. (also less worry about migration if/when the `fabric` mod id becomes `fabric-api` or similar)

Reply:
Fabric tags do use `fabric` as the namespace already (tool tags like `#fabric:axes`).